### PR TITLE
Eng 2336 add Code of Conduct

### PR DESCRIPTION
**Pull request by go-bears**
Adding new Code of Conduct vue page
updated routes.js to point to new page
added page to Secondary Menu

### CODE REVIEW

**Comment by go-bears** [comment]
@ksekhri are we writing unit tests for new pages or that data on pages exist, like the one we broke when removing API Status?  I'm only seeing unit tests for components and util folders.

File: src/pages/Build/CodeOfConductPage.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+		DeveloperSecondaryMenu,
+	},
+	metaInfo: {
+		title: 'Code of Conduct'
+	}
+};
+</script>
+
+<style lang="scss">
+@import 'settings';
```
Comment (mcstover) [inline src/pages/Build/CodeOfConductPage.vue]: You'll only need this if you are using variable in the same style element. Not a biggie.

**Comment by mcstover** [comment]
@go-bears No need for tests around static html content.

File: src/pages/Build/CodeOfConductPage.vue (hunk 1 above)
Comment (go-bears) [reply src/pages/Build/CodeOfConductPage.vue]: Sure thing. But it's a good idea to remove Line 131 to keep the file clean?

**Comment by go-bears** [comment]
woot! it was hard to find those 2 darn spaces. lol!

**Pull request merged**